features/cli/build.feature

```diff
@@ -23,9 +23,9 @@ Feature: build 'apps' with CLI
     Then the output should contain:
       | ruby-hello-world |
     When I run the :new_build client command with:
-      | app_repo |  openshift/ruby:2.0~https://github.com/openshift/ruby-hello-world.git |
-      | strategy | docker                                                                |
-      | name     | n1                                                                    |
+      | app_repo |  openshift/ruby:latest~https://github.com/openshift/ruby-hello-world.git |
+      | strategy | docker                                                                   |
```

Comment:
This docker build will not fail?

Reply:
not fail

Reply:
lgtm when @xiuwang is happy

Reply:
LGTM

Reply:
Build not fail? use ruby:25 with ruby-hello-world to do docker build?

Reply:
seems like this comment needs addressing

Reply:
yes, this needs to be addressed.

Reply:
This has been addressed.